internal/rust/scanner/src/lib.rs

```diff
@@ -515,6 +515,7 @@ a
 
     // TODO(affo): this fails.
     #[test]
+    #[ignore] // See https://github.com/influxdata/flux/issues/1448
```

Comment:
I think there is another test that fails in the parser.

The duration one

Reply:
Correct, it has also been ignored https://github.com/influxdata/flux/pull/1455/files#diff-d998de313c2d6e6df4ca4916fc1d4da0R5029

Reply:
Wops, didn't see that!